internal/testutil/testutil.go

```diff
@@ -88,11 +86,11 @@ func testContext() (Context, error) {
 		return tc, noProjectID
 	}
 
-	pkg, err := build.Import("github.com/GoogleCloudPlatform/golang-samples", "", build.FindOnly)
-	if err != nil {
-		return tc, fmt.Errorf("Could not find golang-samples on GOPATH: %v", err)
-	}
-	tc.Dir = pkg.Dir
+	// pkg, err := build.Import("github.com/GoogleCloudPlatform/golang-samples", "", build.FindOnly)
+	// if err != nil {
+	// 	return tc, fmt.Errorf("Could not find golang-samples on GOPATH: %v", err)
+	// }
+	tc.Dir = "."
```

Comment:
This code blocks a Cloud Run sample using go.mod from working. Not sure what should be done as part of this PR, if anything.

Reply:
`Dir` is used by the GAE E2E tests.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/internal/e2e/standard_test.go#L35 calls `tc.Path`, which uses `tc.Dir` (https://github.com/GoogleCloudPlatform/golang-samples/blob/master/internal/testutil/testutil.go#L36).

If we can get rid of that `GOPATH` logic, lets go for it. But, unfortunately we can't just delete it/comment it out.

Reply:
Alright, I've expanded scope on this PR to try fixing it, as without a fix this PR is blocked. Pushed an update just now with a proposed use of golang.org/x/tools/go/packages as suggested in https://github.com/golang/go/issues/26433, let's see if it works.